modules/ROOT/pages/dynamic-expression-field.adoc

```diff
@@ -128,11 +128,10 @@ After you post the message, return to your flow in Design Center, open the log,
 
 === What to do next
 
-Try adding a Set Variable card after the HTTP Listener card, creating a variable that stores the value of `payload.firstName`, and then changing the message in the Logger to use the variable instead of the payload element. In the Logger card, the variable will appear in the list on the right. You can delete the payload element from the message and then drag in the variable.
+Try adding a Set Variable card after the HTTP Listener card. When data is output from a card, it's good practice to save the data in a variable, so that you can use the data later in your flow. Create a variable that stores the value of `payload.firstName`, and then change the message in the Logger to use the variable instead of the payload item. In the Logger card, the variable will appear in the list on the right. You can delete the payload item from the message and then drag in the variable.
```

Comment:
Why not use "Add a Set Variable card' instead of "Try adding"?

"it's good practice" needs an "a" between "good" and "practice"

"the variable will appear > "the variable appears"

Reply:
"Why not...": because I'm offering a suggestion.

"'It's good practice...'": so, "it's good a practice"? Thank you for the suggestion, but I disagree.

"'the variable will appear'...": I don't believe that using the future tense is a problem here.